schedule/yast/maintenance/create_hdd_transactional_server_dev.yaml

```diff
@@ -14,9 +14,7 @@ schedule:
     - installation/setup_libyui
     - installation/product_selection/install_SLES
     - installation/licensing/accept_license
-    - installation/registration/register_via_scc
-    - installation/module_registration/register_module_transactional
-    - installation/add_on_product_installation/add_additional_products
+    - '{{additional_fist}}'
```

Comment:
let's create two files instead of using conditional logic, in the future we can think about how to solve this with defaults, but it is too soon for maintenance IMO, also I don't know what you wanted to write with 'fist' :)

Reply:
changed.